src/App.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
-import Calculator from './components/Calculator';
+// import Calculator from './components/Calculator';
```

Comment:
- [ ] It will be nice to remove commented code from your code base, as this makes your code base dirty and less readable.